pkgs/games/openxray/default.nix

```diff
@@ -35,25 +35,30 @@ let
     };
   };
 in stdenv.mkDerivation rec {
-  pname = "OpenXRay";
+  pname = "openxray";
   inherit version src;
 
-  hardeningDisable = [ "format" ];
-  cmakeFlags = [ "-DCMAKE_INCLUDE_PATH=${cryptopp}/include/cryptopp" ];
-
+  # TODO https://github.com/OpenXRay/GameSpy/pull/6, check if merged in version > 822
+  # Fixes format hardening
   patches = [
     (fetchpatch {
-      url = "https://github.com/OpenXRay/xray-16/commit/4532cba11e98808c92e56e246188863261ef9201.patch";
-      sha256 = "1hrm4rkkg946ai95krzpf3isryzbb2vips63gxf481plv4vlcfc9";
+      url = "https://github.com/OpenXRay/GameSpy/pull/6/commits/155af876281f5d94f0142886693314d97deb2d4c.patch";
```

Comment:
```suggestion
      url = "https://github.com/OpenXRay/GameSpy/commit/155af876281f5d94f0142886693314d97deb2d4c.patch";
```

Reply:
I don't like that format, it requires an extra comment with a link to the PR to properly track the merge status. The ".patch"-less commit link just tells you "this commit doesn't belong to any branch on this repository".